isis/src/base/apps/shadow/shadow.xml

```diff
@@ -170,6 +173,7 @@
       </parameter>
       <parameter name="TO">
         <type>cube</type>
+        <pixelType>real</pixelType>

```

Comment:
This is really the only required change for this?

Reply:
Yes, the change was recommended by the author of the ticket and fixed the bug.